pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -113,19 +114,21 @@ func gcObjects(ctx *volumemgrContext, dirName string) {
 		return
 	}
 	for _, location := range locations {
-		volumeName := strings.Split(location.Name(), ".")
-		if len(volumeName) > 1 {
-			volumeKey := volumeName[0]
-			vs := lookupVolumeStatus(ctx, volumeKey)
-			if vs == nil {
-				log.Errorf("gcObjects: Found unused volume %s in %s. Deleting it.",
-					location.Name(), dirName)
-				deleteFile(path.Join(dirName, location.Name()))
+		filelocation := path.Join(dirName, location.Name())
+		key, format, err := getVolumeKeyAndFormat(dirName, location.Name())
+		if err != nil {
+			log.Error(err)
+			deleteFile(filelocation)
```

Comment:
Would be nicer to make the log entry say that it is deleting it (because if the bad form of filename).

Reply:
Or does deleteFile do a log.Info?

Reply:
It makes sense to add log.Info in deleteFile